src/pages/LoginPage/LoginPage.tsx

```diff
@@ -14,13 +13,7 @@ const LoginPage: FC = () => {
       <div className={styles["login-page"]}>
         <LoginForm />
         <Delimiter />
-        <AuthBy
```

Comment:
AuthBy does not show the needed component anymore somehow 
![image](https://github.com/ArtemBohak/Chapter/assets/115377006/33c0887d-504d-4581-9cbb-092556a26444)
![image](https://github.com/ArtemBohak/Chapter/assets/115377006/3d643e2c-eedd-4761-a088-f9aab57dd241)